Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMHTMLFrameElement.cpp

```diff
@@ -489,22 +483,4 @@ WebKitDOMDOMWindow* webkit_dom_html_frame_element_get_content_window(WebKitDOMHT
     return WebKit::kit(item->contentWindow());
 }
 
-glong webkit_dom_html_frame_element_get_width(WebKitDOMHTMLFrameElement* self)
-{
-    WebCore::JSMainThreadNullState state;
-    g_return_val_if_fail(WEBKIT_DOM_IS_HTML_FRAME_ELEMENT(self), 0);
-    WebCore::HTMLFrameElement* item = WebKit::core(self);
-    glong result = item->width();
-    return result;
-}
-
-glong webkit_dom_html_frame_element_get_height(WebKitDOMHTMLFrameElement* self)
-{
-    WebCore::JSMainThreadNullState state;
-    g_return_val_if_fail(WEBKIT_DOM_IS_HTML_FRAME_ELEMENT(self), 0);
-    WebCore::HTMLFrameElement* item = WebKit::core(self);
-    glong result = item->height();
-    return result;
-}
```

Comment:
Here you can change the functions to do nothing:

```
glong webkit_dom_html_frame_element_get_width(WebKitDOMHTMLFrameElement* self)
{
    g_return_val_if_fail(WEBKIT_DOM_IS_HTML_FRAME_ELEMENT(self), 0);
    return 0;
}
```

Reply:
Done.